pkg/lakom/resolvetag/admission.go

```diff
@@ -58,7 +58,7 @@ func (hb HandleBuilder) WithLogger(logger logr.Logger) HandleBuilder {
 }
 
 // Build builds a handler from the HandleBuilder.
-func (hb HandleBuilder) Build(ctx context.Context) (*handler, error) {
+func (hb HandleBuilder) Build(_ context.Context) (*handler, error) {
```

Comment:
As the method argument is not used, it can be completely removed.

```suggestion
func (hb HandleBuilder) Build() (*handler, error) {
```

Reply:
Please check that, seems to be not yet resolved.

Reply:
Oops, pushed another commit.